app/models/dfe_sign_in/user_data_importer.rb

```diff
@@ -0,0 +1,15 @@
+class DfeSignIn::UserDataImporter
+  def run
+    users = DfeSignIn::Api::User.all
+    users.each do |u|
+      user = DfeSignIn::User.find_or_initialize_by(dfe_sign_in_id: u.user_id)
+
+      user.given_name = u.given_name
+      user.family_name = u.family_name
+      user.email = u.email
+      user.organisation_name = u.organisation_name
+
+      user.save
```

Comment:
We should probably call `save!` so this fails noisily if unexpectedly a user record cannot be saved or updated, otherwise we'll silently fail to update/save records.